src/components/Notification/Stories/basicNotificationDrawerStory.js

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import { boolean } from '@storybook/addon-knobs';
+import { inlineTemplate } from '../../../../storybook/decorators/storyTemplates';
+import { DOCUMENTATION_URL } from '../../../../storybook/constants';
+import { Notification } from '../index';
+import { NotificationDrawer } from '../NotificationDrawer/index';
+import { Icon } from '../../Icon';
+import { MenuItem } from '../../MenuItem';
+import { Button } from '../../Button';
+
+const handleClick = e => {
+  e.preventDefault();
+  action('some on click')();
+};
+const expandedBool = boolean('Expanded Drawer', false);
+const basicNotificationDrawerStory = stories => {
+  stories.addWithInfo('Notification Drawer', '', () => {
+    const story = (
+      <nav className="navbar navbar-pf-vertical">
+        <nav className="collapse navbar-collapse">
+          <ul className="nav navbar-nav navbar-right navbar-iconic">
+            <li className="drawer-pf-trigger open">
+              <a className="nav-item-iconic" onClick={handleClick}>
+                <span className="fa fa-bell" title="Notifications" />
+              </a>
+            </li>
+          </ul>
+        </nav>
+        <NotificationDrawer expanded={expandedBool}>
+          <NotificationDrawer.Title id="1" title="Notifications" />
+          <NotificationDrawer.Accordion style={{ overflowY: 'auto' }}>
+            <NotificationDrawer.Panel>
+              <NotificationDrawer.PanelHeading>
+                <NotificationDrawer.PanelTitle>
+                  <a onClick={handleClick} aria-expanded="true">
+                    Notification Tab 1
+                  </a>
+                </NotificationDrawer.PanelTitle>
+                <NotificationDrawer.PanelCounter text="3 New Events" />
```

Comment:
@gilad215 the [Notification Drawer Design Pattern](http://www.patternfly.org/pattern-library/communication/notification-drawer/#design) was recently updated to status "# Unread Events" rather than "# New Events" in the accordion.  Can you please update this?